src/components/vertical-divider/vertical-divider.stories.mdx

```diff
@@ -0,0 +1,371 @@
+      <Icon type='info' />
+      <VerticalDivider displayInline />
+      <Icon type='warning' />
+      <VerticalDivider displayInline />
+      <Icon type='chevron_up' />
+      <VerticalDivider displayInline />
+      <Icon type='chevron_down' />
+      <VerticalDivider displayInline />
+      <Icon type='edit' />
+    </div> 
+  </Story>
+</Preview>
+
+### With custom spacing and height
+The `p` prop allows for the spacing on all sides to be customised. The `pt` and `pb` props set the vertical spacing and 
+the `pr` and `pr` props set the horizontal spacing. These props can be number mulitipliers of the base 
+theme spacing which is `8px`, or any valid CSS string. The default for the `p` prop is `3` and therefore `24px`.
+
+<Preview>
+  <Story name='with custom spacing and height'>
+    <div style={{ display: 'inline-flex' }}> 
+      <div>
+        <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} /> 
+        <VerticalDivider h='100px' pt={1} pb={1} />
+        <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} />
+      </div>
+      <VerticalDivider pl={1} pr={1} h={185} />
+      <div>
+        <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} /> 
+        <VerticalDivider h='100px' pt={2} pb={2} />
+        <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} />
+      </div>
+      <VerticalDivider pl={2} pr={2} h={200} />
+      <div>
+        <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} /> 
+        <VerticalDivider h='100px' pt={3} pb={3} />
+        <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} />
+      </div>
+      <VerticalDivider pl={3} pr={3} h={215}/>
+      <div>
+        <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} /> 
+        <VerticalDivider h='100px' pt={4} pb={4} />
+        <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} />
+      </div>
+      <VerticalDivider pl={4} pr={4} h={230} />
+      <div>
+        <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} /> 
+        <VerticalDivider h='100px' pt={5} pb={5} />
+        <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} />
+      </div>
+      <VerticalDivider pl={5} pr={5} h={245} />
+      <div>
+        <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} /> 
+        <VerticalDivider h='100px' pt={6} pb={6} />
+        <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} />
+      </div>
+      <VerticalDivider pl={6} pr={6} h={260} />
+      <div>
+        <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} /> 
+        <VerticalDivider h='100px' pt={7} pb={7} />
+        <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} />
+      </div>
+    </div>
+  </Story>
+</Preview>
+
+### Tint variants
+It is possible to pass a value (one of `20`, `75`, `80`, `90`) to the `tint` prop (default is `80`) to adjust the tint 
```

Comment:
This line probably needs updating now that `tint` is no longer restricted to just these values.